integration/test/metric_test.go

```diff
@@ -138,3 +142,14 @@ func readLabel(sample string, i int) (string, int) {
 	end := i + equalsIndex
 	return sample[i:end], end
 }
+
+func shouldSkipMetric(dupMetric string) bool {
+	// Skip metrics that are belongs to aggr_efficiency template as Rest collector don't have separate template
+	skip := []string{"logical_used_wo_snapshots", "logical_used_wo_snapshots_flexclones", "physical_used_wo_snapshots", "physical_used_wo_snapshots_flexclones", "total_logical_used", "total_physical_used"}
+	for _, s := range skip {
+		if strings.Contains(dupMetric, s) {
```

Comment:
Agreed. Start with exact and relax if needed. That means this check can change to a map instead of searching a slice.

These are the metrics in question
```
aggr_logical_used_wo_snapshots
aggr_logical_used_wo_snapshots_flexclones
aggr_physical_used_wo_snapshots
aggr_physical_used_wo_snapshots_flexclones
aggr_total_logical_used
aggr_total_physical_used
```

Reply:
Moved the check to above where key as `aggr_total_logical_used{aggr=\"astra_aggr2\",cluster=\"A250-41-42-43\",datacenter=\"Rest\",node=\"A250-42\"}`  and now `HasPrefix` would be used instead `contains`.